packages/StepTracker/StepTracker.modules.scss

```diff
@@ -0,0 +1,112 @@
+@import '~@tds/core-colours/colours';
+@import '~@tds/core-responsive/responsive';
+
+.container {
+  padding: 0;
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem -1.5rem;
+
+  @include mq($from: md) {
+    margin: 1rem 0;
+  }
+}
+
+.step {
+  width: 100%;
+  position: relative;
+  text-align: center;
+  list-style: none;
+
+  &::before,
+  &::after {
+    position: absolute;
+    top: 17px;
+    width: calc(50% - 1.07rem);
+    content: ' ';
+    border-bottom: 0.1rem solid $color-shuttle-grey;
+  }
```

Comment:
What is going on here with the before and after pseudo elements? Lots of scary hard coded numbers here.

Reply:
This controls the lines that appear before and after each step. That calc makes sure that each line is responsive, but also stays short enough to not go through the circles. I played around with trying a better solution earlier but moved on to higher priority things, I can probably revisit this once I'm done with the more important changes.